mmdet3d/models/segmentors/base.py

```diff
@@ -63,18 +69,18 @@ def forward(self, return_loss=True, **kwargs):
     def show_results(self,
                      data,
                      result,
-                     palette=None,
                      out_dir=None,
+                     palette=None,
```

Comment:
This was originally a bug because `test.py` actually calls this `show_results()` function in an order that first inputs `out_dir` then `palette`.

Reply:
Ah, I think I can explicitly assign values to `out_dir`, so that the order won't make any difference.